src/app/auth/login/page.tsx

```diff
@@ -0,0 +1,72 @@
+'use client'
+import login from '@/app/api/auth/login';
+import React, { useState } from 'react';
+import { Button, Checkbox, Form, Input } from 'antd';
+
+export default function Page() {
+    const onFinish = async ( values: any ) => {
+        console.log( 'Success:', values );
+        const userData = {
+            email: values.email,
+            password: values.password
+        }
+        await login( userData );
+
+    };
+
+    const onFinishFailed = ( errorInfo: any ) => {
+        console.log( 'Failed:', errorInfo );
+    };
+
+    type FieldType = {
```

Comment:
I think that you can rename this type with a more descriptive name, for example: RegisterType.